processors/minsev/minsev.go

```diff
@@ -8,20 +8,28 @@ package minsev // import "go.opentelemetry.io/contrib/processors/minsev"
 import (
 	"context"
 
-	api "go.opentelemetry.io/otel/log"
 	"go.opentelemetry.io/otel/sdk/log"
 )
 
 // NewLogProcessor returns a new [LogProcessor] that wraps the downstream
 // [log.Processor].
 //
+// severity reports the minimum record severity that will be logged. The
+// LogProcessor discards records with lower severities. If severity is nil,
+// SeverityInfo is used as a default. The LogProcessor calls severity.Severity
+// for each record processed or queried; to adjust the minimum level
+// dynamically, use a [SeverityVar].
+//
 // If downstream is nil a default No-Op [log.Processor] is used. The returned
 // processor will not be enabled for nor emit any records.
-func NewLogProcessor(downstream log.Processor, minimum api.Severity) *LogProcessor {
+func NewLogProcessor(downstream log.Processor, severity Severitier) *LogProcessor {
 	if downstream == nil {
 		downstream = defaultProcessor
 	}
-	p := &LogProcessor{Processor: downstream, Minimum: minimum}
+	if severity == nil {
+		severity = SeverityInfo
```

Comment:
This what would be used in the default mentioned in the example. In which case, shouldn't this be `SeverityUndefined`?

Reply:
I don't follow. The default in the example is `log.SeverityInfo`. This is documented to default to `log.SeverityInfo`. We want to default to a usable value when we can, and that is what it is doing here. I'm not sure why you wan this to be `SeverityUndefined`.